angular/src/app/viewer/visualization/network-display/network-display.component.ts

```diff
@@ -88,56 +87,51 @@ export class NetworkDisplayComponent implements AfterViewInit {
 
   ngAfterViewInit(): void {
     console.log('Values at NetDisplayComp on ngAfterViewinit():');
-    console.log('layoutPicker: ', this.layoutPicker);
   }
```

Comment:
I think this is the `console.log` I put in, right? I think it would be fine to get rid of it (along with the `AfterViewInit` implementation), unless you want to keep it in a little longer for debugging purposes.